tensorflow/core/kernels/gpu_utils.h

```diff
@@ -93,15 +92,26 @@ class AutotuneMap {
   bool Find(const Parameters& params, Config* config) const {
     mutex_lock lock(mu_);
     auto iter = params_config_map_.find(params);
-    if (iter == params_config_map_.end() ||
-        (iter->second.score < min_score_threshold_ &&
-         iter->second.count <= max_autotune_count_)) {
+
+    if (iter == params_config_map_.end()) {
       return false;
     }
     *config = iter->second.config;
     return true;
   }
+
   void Insert(const Parameters& params, const Config& config) {
+    mutex_lock lock(mu_);
+    auto iter = params_config_map_.find(params);
+    if (iter == params_config_map_.end()) {
+      // Create a new entry if params is new.
+      VLOG(1) << GetActionSummary("creates", params, config);
+      params_config_map_.insert(
+          std::make_pair(params, ValueType{config, 1, 1}));
+    }
+  }
+
+  void InsertBasedOnScore(const Parameters& params, const Config& config) {
```

Comment:
Does any code ever call this newly added function?

Reply:
Not currently. The PR originally integrated cuBLASLt into native TF and XLA, before the TF part was removed following George's advice. I'll submit a PR for the second part which will make use of this function once the current PR has been merged. If it's a concern, I'll remove the function.

Reply:
It's fine, you can leave it. I missed that part of the history and was curious.